spec/models/item_spec.rb

```diff
@@ -32,4 +32,16 @@
     expect { create(:item, price: 0) }.to raise_error(ActiveRecord::RecordInvalid)
   end
 
+  it "is invalid with a description over 300 characters" do
+    expect { create(:item, description: "jssssssssssssssssssssssssssssssssss
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.